keps/sig-scheduling/5055-dra-device-taints-and-tolerations/README.md

```diff
@@ -535,8 +676,39 @@ type DeviceTaintSelector struct {
     // +listType=atomic
     Selectors []DeviceSelector
 }
+
+// DeviceTaintRuleStatus provides information about an on-going pod eviction.
+type DeviceTaintRuleStatus struct {
+    // Conditions provide information about the current state of the DeviceTaintRule
+    // in a machine-readable and human-readable format.
+    //
+    // The following condition is currently defined as part of this API, more may
+    // get added:
+    // - Type: EvictionInProgress
+    // - Status: true if there are pods which still need to be evicted, false otherwise
+    //   (includes the effects which don't cause eviction).
+    // - Reason: not specified, may change
+    // - Message: includes information about number of pending pods and already evicted pods
+    //   in a human-readable format, updated periodically, may change
+    //
+    // Must have 8 or less entries.
+    //
+    // +optional
+    // +listType=map
+    // +listMapKey=type
+    Conditions []metav1.Condition
+}
+
+const DeviceTaintRuleStatusMaxConditions = 8
 ```
 
+The condition is meant to be used to check for completion of a triggered eviction with:
+
+    kubectl wait --for=condition=EvictionInProgress=false DeviceTaintRule/my-taint-rule
+
+Reasons are not specified as part of the API because it's not required by the
+use cases and would restrict future changes unnecessarily.
+
 ### Test Plan
```

Comment:
Can we add test plan for the case when multiple rules hit the same device set with different rates?

We may need to add another test case to validate rate limiting and controller back-pressure.